### ISSUE TITLE
Tips listener doing something weird w/ unsupported data types

### ISSUE THREAD

**Issue by oraclown** [tellor-io/telliot-feeds#514]
someone tipped twtr/usd (`0x8f096ae9dbf9d35d36368bc03c0b9843ae6b5ac6ec980d41e99ce61ed4bc903d `) and this happens:
(see "Error getting eligible funded feeds:...")
```console
INFO    | telliot_feeds.reporters.tellor_360 | 
            STAKER INFO
            start date: 1670340607
            stake_balance: 2000.0
            locked_balance: 0
            last report: 1671844761
            reports count: 24
            
{'code': -32000, 'message': 'out of gas'}
Multicall batch size reduced from 8 to 7. The failed batch had 8 calls.
ERROR   | telliot_feeds.reporters.tips.listener.funded_feeds | Error getting feeds, timestamps, and values
ERROR   | telliot_feeds.reporters.tips.listener.funded_feeds | Error getting eligible funded feeds: values_tup not in multiple_values_response: ('values_array', 0x8f096ae9dbf9d35d36368bc03c0b9843ae6b5ac6ec980d41e99ce61ed4bc903d)
INFO    | telliot_feeds.reporters.interval | Unable to suggest datafeed
INFO    | telliot_feeds.reporters.interval | Sleeping for 20 seconds
INFO    | telliot_feeds.reporters.tellor_360 | Current Oracle stakeAmount: 10.0
INFO    | telliot_feeds.reporters.tellor_360 | 
            STAKER INFO
            start date: 1670340607
            stake_balance: 2000.0
            locked_balance: 0
            last report: 1671844761
            reports count: 24
            
{'code': -32000, 'message': 'out of gas'}
Multicall batch size reduced from 7 to 6. The failed batch had 7 calls.
INFO    | telliot_feeds.sources.price_aggregator | Running median on [0.04679573, 0.04719499179831757, 0.04742]
INFO    | telliot_feeds.sources.price_aggregator | Feed Price: 0.04719499179831757 reported at time 2022-12-24 01:40:47.545510+00:00
INFO    | telliot_feeds.sources.price_aggregator | Number of sources used in aggregate: 3
INFO    | telliot_feeds.sources.price_aggregator | Running median on [0.04677987, 0.04719499179831757, 0.04742]
INFO    | telliot_feeds.sources.price_aggregator | Feed Price: 0.04719499179831757 reported at time 2022-12-24 01:40:47.977809+00:00
INFO    | telliot_feeds.sources.price_aggregator | Number of sources used in aggregate: 3
INFO    | telliot_feeds.sources.price_aggregator | Running median on [0.04677987, 0.04719499179831757, 0.04742]
INFO    | telliot_feeds.sources.price_aggregator | Feed Price: 0.04719499179831757 reported at time 2022-12-24 01:40:48.318781+00:00
INFO    | telliot_feeds.sources.price_aggregator | Number of sources used in aggregate: 3
INFO    | telliot_feeds.reporters.tips.listener.funded_feeds | No eligible funded feeds found
INFO    | telliot_feeds.reporters.interval | Unable to suggest datafeed
INFO    | telliot_feeds.reporters.interval | Sleeping for 20 seconds
```
It should provide a msg like "twtr/usd source not available, report manually? instead of brushing some keyerror under the rug lol

**Comment by oraclown** [comment]
@spuddyminer confirms `the error message repeats every cycle and I don't think it's submitting anything`

**Comment by spuddyminer** [comment]
It looks like telliot DOES submit when there are tips available...so maybe the error message is just confusing? Logs attached (still gathering logs)
[telliot_logs.txt](https://github.com/tellor-io/telliot-feeds/files/10315518/telliot_logs.txt)

**Comment by akremstudy** [comment]
Not sure what was causing this but its not showing up anymore.  Added some fixes in #529 to avoid redundant calls and the errors are not showing up but we should keep an eye on it to see if it happens again.

**Comment by spuddyminer** [comment]
I haven't experienced this after recent commits. Closing now
